server/index.js

```diff
@@ -14,72 +14,42 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
-app.use('/:id', express.static('client/dist'));
 
-
-// app.get('/api/products', (req, res) => {
-//   controller.ikea_products.get(req, res, (err, data) => {
-//     if (err) {
-//       // eslint-disable-next-line no-console
-//       console.error('error connecting to server');
-//       return;
-//     }
-//     // eslint-disable-next-line no-console
-//     console.log('ikea_products connected');
-//     res.send(data).status(200);
-//   });
-// });
-
-app.get('/api/products/', (req, res) => {
-  dbOperate.product_readALL((err, data) => {
+app.get('/api/products', (req, res) => {
+  controller.ikea_products.get(req, res, (err, data) => {
```

Comment:
Same question applies here, do you actually want to get every product from your db? This will probably crash something